### ISSUE TITLE
Framework name is not stripped in terminal log

### ISSUE THREAD

**Issue by fi3ework** [rspack-contrib/storybook-rsbuild#8]
The builder name log is verbose as shown below, which is much more verbose than Vite or webpack framework. There's a strip function https://github.com/storybookjs/storybook/blob/a5c7f7fd16643c3aa85fceb9149e3d2e83b9ef7d/code/lib/core-server/src/build-dev.ts#L216-L219 which only works with `@storybook/name` frameworks. 

<img width="677" alt="image" src="https://github.com/rspack-contrib/storybook-rsbuild/assets/12322740/b003df60-2354-4afb-9701-66b19e8ae794">

![img_v3_02av_562a3eef-18b8-456f-ab5d-8ae3f61b7fbg](https://github.com/rspack-contrib/storybook-rsbuild/assets/12322740/437f5a12-00b6-40bc-bef1-4ffcea8fd044)

**Comment by tmkx** [comment]
it shows a full path because the `framework` option is from `getAbsolutePath`. it's okay if we use `framework: 'storybook-react-rsbuild'` instead of `framework: getAbsolutePath('storybook-react-rsbuild')`

![image](https://github.com/rspack-contrib/storybook-rsbuild/assets/13805071/a79cc72d-0f56-4629-b8df-8b88ba2aa18e)

https://github.com/storybookjs/storybook/blob/a5c7f7fd16643c3aa85fceb9149e3d2e83b9ef7d/code/lib/core-server/src/build-dev.ts#L81

**Comment by fi3ework** [comment]
@tmkx 
Yes, it will. However, it's very prone to failed to find addons / builder / frameworks in a pnpm monorepo without a resolved dep path. (see how many `getAbsolutePath` is used in Storybook https://github.com/search?q=repo%3Astorybookjs%2Fstorybook%20getAbsolutePath&type=code)

**Comment by tmkx** [comment]
I mean it works as expected in user's projects. it shows the full path in this repo only.

**Comment by fi3ework** [comment]
It will shows as expected in user project ONLY IF they don't use `getAbsolutePath`. But offical's framework will show the neat name even with `getAbsolutePath`.

**Comment by tmkx** [comment]
i see, this may occur in a monorepo which shares Storybook dependencies. agree with u that it's better to do something like `frameworkName.split(/[\\/]/).at(-1)` in storybook

**Comment by fi3ework** [comment]
close since Storybook has changed the way of welcome logging, this issue is gone.

<img width="715" height="777" alt="Image" src="https://github.com/user-attachments/assets/4bda109a-e994-4749-8ef6-5cd13ca8c99d" />
